### PR TITLE
Add FPX to the list of source types for V4.x

### PR DESCRIPTION
**Pull request by hpariwat**
Add FPX to the list of existing source types.

### CODE REVIEW

**Comment by danfowler** [comment]
Do we also need to update our model for a Payment Method in Capability?

https://github.com/omise/omise-java/blob/master/src/main/java/co/omise/models/PaymentMethod.java

**Review by danfowler — APPROVED** [review]

File: src/test/resources/testdata/objects/capability_object.json [hunk 1]
```diff
@@ -42,7 +42,8 @@
         8,
         9,
         10
-      ]
+      ],
+      "banks": []
```
Comment (rezigned) [inline src/test/resources/testdata/objects/capability_object.json:46]: Can we also have the test cases where `banks` is not empty? So, we can ensure that `Bank` model is serialized correctly.

**Review by rezigned — APPROVED** [review]

**Review by tookaa2 — APPROVED** [review]
